src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html

```diff
@@ -2,65 +2,70 @@
   <div class="edit-space-description-widget" user-level>
     <div class="col-sm-3 f8-space-masthead-space-info">
       <div>
-        <span class="f8-space-masthead-name">{{space?.attributes.name | spaceName}}</span> <span class="f8-space-masthead-owner">bobross_925</span>
+        <span class="f8-space-masthead-name">{{space?.attributes.name | spaceName}}</span>
+        <span class="f8-space-masthead-owner">bobross_925</span>
       </div>
       <div>
         <span class="pficon pficon-users fa-lg"></span> 59
-        <span class="f8-space-masthead-vertical-divider-collab"></span>
-        <span class="pficon pficon-add-circle-o fa-lg"></span> Add Collaborator
+        <span id="space-description-collaborators-placeholder" *ngIf="!userOwnsSpace">Collaborators</span>
+        <ng-container *ngIf="userOwnsSpace" id="edit-space-description-add-collaborator-btn">
+          <span id="user-level-collaborator-divider" class="f8-space-masthead-vertical-divider-collab"></span>
+          <span id="user-level-add-collaborator" class="pficon pficon-add-circle-o fa-lg"></span> Add Collaborator
+        </ng-container>
       </div>
     </div>
-    <div class="f8-space-masthead-vertical-divider"></div>
-    <div class="col-sm-9 f8-space-masthead-description">
-      <!-- There must be no spaces here, otherwise we end up with random whitespace in the editable-->
-      <div class="form-group">
-        <div class="f8-space-masthead-description-title"></div>
-        <textarea row="3"
-                  placeholder="Description goes here"
-                  class="space-description form-control"
-                  *ngIf="isEditing && (isEditable() | async)"
-                  (keyup.enter)="onUpdateDescription(description.value)"
-                  (keydown.enter)="preventDef($event)"
-                  #description>{{space?.attributes.description}}</textarea>
-        <div class="f8-space-masthead-description-edit-buttons" *ngIf="isEditing">
-          <button id="_btn_cancel_description"
-                  class="pull-right btn btn-default edit-space-description-btn"
-                  (click)="stopEditingDescription()">
-            Cancel
-          </button>
-          <!--Save icon-->
-          <button id="workItemTitle_btn_save_description"
-                  class="pull-right btn btn-primary edit-space-description-btn "
-                  (click)="saveDescription()" >
-            Save
-          </button>
+    <ng-container *ngIf="userOwnsSpace; else uneditableDescription">
```

Comment:
Most of the functionality for this component has been wrapped with ng-container. This could be another candidate for an alternate, more simplified layout.